core/templates/services/angular-services.index.ts

```diff
@@ -450,6 +451,7 @@ export const angularServices: [string, Type<{}>][] = [
   ['FatigueDetectionService',
     FatigueDetectionService],
   ['FeedbackThreadObjectFactory', FeedbackThreadObjectFactory],
+  ['FilterForMatchingSubstringPipe', FilterForMatchingSubstringPipe],
```

Comment:
I don't think this is required. Just adding pipe to module is sufficient.

Reply:
This seems to be an index of all Angular services / controllers / utilities so it may be needed -- @srijanreddy98 @darksun27 WDYT?

Reply:
If the pipe is an Injectable, it has to be added here.

Reply:
Ack, leaving it in